client-app/shared/static-content/components/text-block.vue

```diff
@@ -1,7 +1,11 @@
 <template>
   <div class="pt-6 pb-16 text-block">
     <div class="w-full max-w-screen-2xl mx-auto px-5 md:px-12">
-      <h1 class="text-2xl mb-8">{{ model.header }}</h1>
+      <h2 v-if="model.heading == 'h2'" class="text-2xl">{{ model.title }}</h2>
+      <h3 v-if="model.heading == 'h3'" class="text-2xl">{{ model.title }}</h3>
+      <h4 v-if="model.heading == 'h4'" class="text-2xl">{{ model.title }}</h4>
+      <h5 v-if="model.heading == 'h5'" class="text-2xl">{{ model.title }}</h5>
+      <h6 v-if="model.heading == 'h6'" class="text-2xl">{{ model.title }}</h6>
```

Comment:
can be on one line:
`<component :is="model.heading" class="text-2xl">{{ model.title }}</component>`

Reply:
fixed